Resources/Textures/Objects/Weapons/Guns/Ammunition/Magazine/Magnum/magnum_smg_mag.rsi/meta.json

```diff
@@ -5,7 +5,7 @@
         "y": 32
     },
     "license": "CC-BY-SA-3.0",
-    "copyright": "https://github.com/discordia-space/CEV-Eris/raw/831c7c2743530e8a31910d781bae9dc34f3f1ec8/icons/obj/ammo_mags.dmi",
+    "copyright": "https://github.com/shiptest-ss13/Shiptest/commit/662c08272acd7be79531550919f56f846726eabb",
```

Comment:
try linking the dmi at the commit (like it was previously) do this for every copyright in the pr

Reply:
all copyright for items subjected to my PR have been changed to include the link to the dmi as well.